src/test/java/htsjdk/samtools/cram/CRAIEntryTest.java

```diff
@@ -14,27 +16,23 @@
  * Created by vadim on 25/08/2015.
  */
 public class CRAIEntryTest extends HtsjdkTest {
-
     @Test
     public void testFromContainer() {
         final Container container = new Container();
-        final Slice slice = new Slice();
-        slice.sequenceId = 1;
-        slice.alignmentStart = 2;
-        slice.alignmentSpan = 3;
-        slice.containerOffset = 4;
```

Comment:
I removed this field from Slice because it was redundant to `Container.offset` and I believe it got out of sync in some cases.